src/lib/executeSourceTransfer.js

```diff
@@ -2,10 +2,18 @@
 
 const log = require('../common').log.create('executeSourceTransfer')
 const validator = require('./validate')
+const moment = require('moment')
+const _ = require('lodash')
+const promiseRetry = require('promise-retry')
+const retryOpts = { // 22 retries over approx. 24 hours
+  retries: 22,
```

Comment:
You can actually use the `forever: true` option: https://github.com/tim-kos/node-retry#retryoperationoptions